pkg/template/controller/metrics.go

```diff
@@ -8,25 +8,34 @@ import (
 	kapi "k8s.io/kubernetes/pkg/api"
 )
 
-var templateInstancesTotal = prometheus.NewGaugeVec(
+var templateInstanceInfo = prometheus.NewGaugeVec(
 	prometheus.GaugeOpts{
-		Name: "openshift_template_instance_total",
-		Help: "Counts TemplateInstance objects by condition type and status",
+		Name: "openshift_template_instance_info",
+		Help: "Information about templateinstance.",
```

Comment:
i know this is copied from KSM but i just want to call @smarterclayton's attention to it because it seems like an unhelpful name+description for a metric that is really "total number of template instances".

Reply:
Lack of sleep may be impairing my judgment, but I'll say it:, KSM is just wrong IMO in this regard; it should end in `_total`